uktrade/data-hub-frontend
# Cls2 543 task tab visibility (#6257)

* Added my-task tab

* WIP: Implementing My Tasks

* WIP: my tasks API call

* Task moved up a level

* Wired up redux and renamed

* Add task count dashboard condition

* Test added plus basic sandbox endpoint

* Migrate logiv to personalised dashboard

* Update dashboard-spec.js

to change test When My Tasks returns no results to 'null'

* PR comment fixes

* Passing a boolean for the count check

* Back to using adviserId

* Configured indext and task to the correct pattern of passing adviser through

* Update mytasks.json

* Rename mytasks.json to myTasks.json

capitalise file name

* remove file

* Test fixed

## Description of change

_Document what the PR does and why the change is needed_

## Test instructions

_What should I see?_

## Screenshots

### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
